docs/front-end/streamfield-blocks.md

```diff
@@ -0,0 +1,8 @@
+# Streamfield blocks
+
+We want our streamfield blocks to be as flexible as possible and re-usable on different page types. Therefore when styling the blocks please apply the following principles:
+
+- Ensure consistent spacing between blocks
```

Comment:
Suggestion: could mention using the `$spacer` vars here https://github.com/torchbox/torchbox.com/blob/integration/sprint-1/tbx/static_src/sass/config/_variables.scss#L177 (not yet in `main`)